tests/test_core.py

```diff
@@ -209,6 +209,22 @@ def test_to_dataset_dict_chunking_2(chunks, expected_chunks):
     assert ds['SHF'].data.chunksize == expected_chunks
 
 
+keys = [
+    'CMIP.CCCma.CanESM5.historical.*.Ofx.*.gn.*',
+    'CMIP.CCCma.CanESM5.historical.r1i1p1f1.Ofx.deptho.gn.*',
+    'CMIP.CCCma.CanESM5.historical',
+    'CMIP.CCCma.CanESM5.historical.r1i1p1f1.Ofx.deptho.gn',
+    'DCPP',
+]
+
+
+@pytest.mark.parametrize('key', keys)
+def test_get_item(key):
+    col = intake.open_esm_datastore(zarr_col_pangeo_cmip6)
+    assert isinstance(col.df, pd.DataFrame)
+    assert len(col) >= 1
```

Comment:
This doesn't actually seem to test `__getitem__`.  Where is `key` used?

Reply:
I updated the tests.